cli/data/get.go

```diff
@@ -27,6 +27,21 @@ import (
 	"strings"
 )
 
+type S3Target struct {
+	bucket string
+	prefix string
+	path string
+}
+
+func (s *S3Target) copy() *S3Target {
```

Comment:
is this here to ensure that we aren't mutating?

Reply:
I don't understand enough yet about Go to grasp if I should be just passing structs by value and allow the runtime to clone those from my understanding but yet.. here was just do that scenario where you want to be a good citzen:

```ruby
class A
  def initialize(options = {})
     opts = options.dup
     opts.merge(...)
  end
end
```

Reply:
Yeah I wouldn't have this function, if anything we can just copy below; and `copy` is a built-in so it's not a great name. I would just pass by value, no reason to pass by reference here.